v3/cypress/e2e/text.spec.ts

```diff
@@ -23,6 +23,9 @@ context("Text tile", () => {
     c.getComponentTitle(kTextTileTestId).should("have.text", textDefaultTitle)
 
     // Redo title change
+    cy.wait(300) // this test has become flaky.
+    // Added a small wait here to see if this fixes
+    // the disabled redo button issue
```

Comment:
Rather than adding an arbitrary timed wait, it would be preferable to wait on what we actually care about:
```suggestion
    toolbar.getRedoTool().should("be.enabled")
```